### ISSUE TITLE
Whitespace recommendation for LESS/SASS operators

### ISSUE THREAD

**Issue by gauthierm** [mdo/code-guide#129]
Should operators (`+`, `-`, etc) have whitespace or no whitespace in style values? Based on the comma whitespace recommendation for color values, I'm leaning towards no whitespace, but some clarity would be nice.

With whitespace:

``` css
  width: @gutter * 2 + 100px;
  margin: 0 1px @vertical-margin * 2 10px;
```

With no whitespace:

``` css
  width: @gutter*2+100px;
  margin: 0 1px @vertical-margin*2 10px;
```

**Comment by mdo** [comment]
Whitespace and parens are super helpful:

``` scss
.element {
  width: (@gutter * 2) + 100px;
  margin: 0 1px (@vertical-margin * 2) 10px;
}
```

They work together super nicely to define natural boundaries for your eyes when skimming code.

**Comment by sgarrity** [comment]
> Whitespace and parens are super helpful:

Agreed. Would you consider adding this to the guide?

**Comment by mdo** [comment]
Yup! Haven't had a chance to update the guide recently, but if anyone wanted to see this added sooner rather than later, PRs are most welcome :).

**Comment by sgarrity** [comment]
PR'ed! Feedback on wording is welcome.

**Comment by mdo** [comment]
Fixed in #131.
